index.html

```diff
@@ -51,154 +72,186 @@ <h1 class="main-header">Hey There. I'm Zelalem</h1>
         alt=""
         class="illustration-header-mobile"
       />
+      <img
+        src="img/Illustration-header-desktop.png"
+        alt="healine illustration"
+        class="illustration-header-desktop"
+      />
```

Comment:
i suggest you use background image rather than img tag, i think it is less code in the html file and you don't need to add css styling for the img